code/xcvm/cosmwasm/contracts/gateway/src/interpreter.rs

```diff
@@ -18,7 +18,7 @@ use crate::{auth, prelude::*};
 pub(crate) fn force_instantiate(
 	_: auth::Admin,
 	gateway: Addr,
-	deps: DepsMut,
+	deps: &mut DepsMut,
```

Comment:
Is this the standard?

Reply:
you mean? what part of it? i just did config batch simplest possible way from config messages so can config and reconfig whole xcvm using one json tx

Reply:
Alternative would be to use `deps.fork()` which clones the DepsMut.  That’s probably the intended way to deal with situations like this but really there’s probably very little difference.

Reply:
I'm more asking about the `&mut` on DeptsMut
https://github.com/deus-labs/cw-contracts/blob/main/contracts/cw-quadratic-funding/src/contract.rs#L18

Reply:
i fixed, used `fork` as suggested michal, i guess that is standard